api/app/apps.proto

```diff
@@ -119,7 +119,7 @@ service Apps {
     */
   rpc ListKustomizations(ListKustomizationsReq) returns (ListKustomizationsRes) {
     option (google.api.http) = {
-      get : "/v1/namespace/{namespace}/app/{app_name}/kustomization"
+      get : "/v1/kustomizations"
```

Comment:
Nit: should this be plural?  Usually the REST semantics are /v1/kustomization, I would expect a list.  Then /v1/kustomization/:id, I would expect to get one.

Reply:
Sure, but not sure why we would try to min/max this as a REST API. I would hope someone would use the open API output to generate a client library in their language of choice.

We already gen the `go` and `ts` clients. I didn't have to change anything in my "user space" code to accommodate this change.

Reply:
https://github.com/weaveworks/weave-gitops/issues/1465